modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java

```diff
@@ -11,20 +11,24 @@
 import com.mongodb.client.TransactionBody;
 import org.bson.Document;
 import org.junit.Test;
+import org.testcontainers.utility.DockerImageName;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
 
 public class MongoDBContainerTest {
+
+    private static final DockerImageName MONGO_IMAGE = new DockerImageName("mongo:4.0.10");
+
     /**
      * Taken from <a href="https://docs.mongodb.com/manual/core/transactions/">https://docs.mongodb.com</a>
      */
     @Test
     public void shouldExecuteTransactions() {
         try (
             // creatingMongoDBContainer {
-            final MongoDBContainer mongoDBContainer = new MongoDBContainer()
+            final MongoDBContainer mongoDBContainer = new MongoDBContainer(MONGO_IMAGE)
```

Comment:
perhaps we should inline the image here, as it will be in the docs

Reply:
Done.